FutureUse.dart

```diff
@@ -45,12 +45,12 @@ Future<String> fetchUsername() async {
 
 Future<void> getUsernameExp() async {
 
-  try{
-    String username = await fetchUsername();
-    print(username);    
-  }catch(e){
-    print(e.toString());
-  }
+	try{
+		String username = await fetchUsername();
+		print(username);    
+	}catch(e){
```

Comment:
In all such cases, brackets should always have space : `try { ... `, `} catch (e) { ...`